### PR TITLE
fix: simplify count statements

### PR DESCRIPTION
**Pull request by DrFaust92**
## Description

Simplify Count statement in 2 cases of policy attachments as it causes failures during apply in some cases

## Motivation and Context

Simplify Count statement in 2 cases of policy attachments as it causes failures during apply in some cases

## Breaking Changes

No.

## How Has This Been Tested?

Yes.

### CODE REVIEW

**Comment by DrFaust92** [comment]
@bryantbiggs if you have some time to take a look.

File: modules/iam-group-with-policies/main.tf [hunk 1]
```diff
@@ -51,7 +51,7 @@ resource "aws_iam_policy" "iam_self_management" {
 }
 
 resource "aws_iam_policy" "custom" {
-  count = length(var.custom_group_policies) > 0 ? length(var.custom_group_policies) : 0
+  count = length(var.custom_group_policies)
```
Comment (bryantbiggs) [inline modules/iam-group-with-policies/main.tf:54]: looks good - might be worth considering a refactor to `for_each` with an array of maps. just food for thought

**Review by bryantbiggs — APPROVED** [review]
looks good to me - @antonbabenko?

File: modules/iam-group-with-policies/main.tf (hunk 1 above)
Comment (DrFaust92) [reply modules/iam-group-with-policies/main.tf:54]: ill go over he module and see where this refactor is needed. i wanted to make changes more incremental

**Comment by antonbabenko** [comment]
Nice and sweet 👍 

[v2.20.0](https://github.com/terraform-aws-modules/terraform-aws-iam/releases/v2.20.0) has been just released.

**Pull request merged**